packages/p2p/src/components/my-profile/payment-methods/payment-methods-list/payment-methods-list.jsx

```diff
@@ -119,66 +124,21 @@ const PaymentMethodsList = () => {
                                             is_only_horizontal
                                             is_scrollbar_hidden
                                         >
-                                            {payment_methods_list.map(
-                                                (each_payment_method, each_payment_method_key) => (
-                                                    <PaymentMethodCard
-                                                        key={each_payment_method_key}
-                                                        payment_method={each_payment_method}
-                                                        small
-                                                        show_payment_method_name={false}
-                                                    />
-                                                )
-                                            )}
+                                            {payment_methods.map((each_payment_method, each_payment_method_key) => (
+                                                <PaymentMethodCard
+                                                    key={each_payment_method_key}
+                                                    payment_method={each_payment_method}
+                                                    small
+                                                    show_payment_method_name={false}
+                                                />
+                                            ))}
                                         </ThemedScrollbars>
                                     </React.Fragment>
                                 );
-                            }
-                        )}
+                            })}
                     </div>
                 </MobileFullPageModal>
             </MobileWrapper>
-            <Modal
```

Comment:
oh man i refactored this modal out in the refactor modals card 👀 https://github.com/farrah-deriv/deriv-app/pull/45/files#diff-1424777e01dad765357cc91c03464a5de740a80866e756a8308179cf550bbfee

Reply:
Hmmm 👀

Reply:
haha its ok, we just need to remember when the p2p-modal-refactoring branch is merged so we can keep track and remove mine later, i also named mine differently hahaha 😆